common/schema.h

```diff
@@ -418,6 +418,8 @@ namespace swss {
 #define CFG_FLOW_COUNTER_ROUTE_PATTERN_TABLE_NAME   "FLOW_COUNTER_ROUTE_PATTERN"
 #define CFG_LOGGER_TABLE_NAME                       "LOGGER"
 
+#define CFG_TX_MONITORS_ARGS_TABLE_NAME "TX_MONITORS_ARGS"
```

Comment:
better to use name that is more clear. for example: TX_ERROR_MONITOR

Reply:
fixed